com.unity.render-pipelines.core/Editor/Camera/CameraUI.Drawers.cs

```diff
@@ -75,89 +68,117 @@ public static void Drawer_Projection(ISerializedCamera p, Editor owner)
 
-                if (!fovAxisVertical && !cam.fovAxisMode.hasMultipleDifferentValues)
-                {
-                    var targets = p.serializedObject.targetObjects;
-                    var camera0 = targets[0] as Camera;
-                    float aspectRatio = isPhysicalCamera ? cam.sensorSize.vector2Value.x / cam.sensorSize.vector2Value.y : camera0.aspect;
-                    // camera.aspect is not serialized so we have to check all targets.
-                    fovCurrentValue = Camera.VerticalToHorizontalFieldOfView(camera0.fieldOfView, aspectRatio);
-                    if (targets.Cast<Camera>().Any(camera => camera.fieldOfView != fovCurrentValue))
-                        multipleDifferentFovValues = true;
-                }
-                else
-                {
-                    fovCurrentValue = cam.verticalFOV.floatValue;
-                    multipleDifferentFovValues = cam.fovAxisMode.hasMultipleDifferentValues;
-                }
+            EditorGUI.BeginChangeCheck();
+            isPhysicalCamera = EditorGUILayout.Toggle(content, isPhysicalCamera);
+            if (EditorGUI.EndChangeCheck())
+                p.projectionMatrixMode.intValue = isPhysicalCamera ? (int)ProjectionMatrixMode.PhysicalPropertiesBased : (int)ProjectionMatrixMode.Implicit;
+            EditorGUILayout.EndHorizontal();
+            EditorGUI.EndProperty();
 
-                EditorGUI.showMixedValue = multipleDifferentFovValues;
-                var content = EditorGUI.BeginProperty(EditorGUILayout.BeginHorizontal(), Styles.fieldOfViewContent, cam.verticalFOV);
-                EditorGUI.BeginDisabledGroup(p.projectionMatrixMode.hasMultipleDifferentValues || isPhysicalCamera && (cam.sensorSize.hasMultipleDifferentValues || cam.fovAxisMode.hasMultipleDifferentValues));
-                EditorGUI.BeginChangeCheck();
-                s_FovLastValue = EditorGUILayout.Slider(content, fovCurrentValue, 0.00001f, 179f);
-                s_FovChanged = EditorGUI.EndChangeCheck();
-                EditorGUI.EndDisabledGroup();
-                EditorGUILayout.EndHorizontal();
-                EditorGUI.EndProperty();
-                EditorGUI.showMixedValue = false;
+            EditorGUI.showMixedValue = false;
+            if (s_FovChanged && (!isPhysicalCamera || p.projectionMatrixMode.hasMultipleDifferentValues))
+            {
+                cam.verticalFOV.floatValue = fovAxisVertical
+                    ? s_FovLastValue
+                    : Camera.HorizontalToVerticalFieldOfView(s_FovLastValue, (p.serializedObject.targetObjects[0] as Camera).aspect);
+            }
+            else if (s_FovChanged && isPhysicalCamera && !p.projectionMatrixMode.hasMultipleDifferentValues)
+            {
+                cam.verticalFOV.floatValue = fovAxisVertical
+                    ? s_FovLastValue
+                    : Camera.HorizontalToVerticalFieldOfView(s_FovLastValue, (p.serializedObject.targetObjects[0] as Camera).aspect);
```

Comment:
I doubt this is ok to work only for the first element in case of multi-selection `targetObjects[0]` Especially when aspect differ